app/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Category;
+
+class CategoryController extends Controller
+{
+    public function create()
+    {
+        return view('Category.CreateCat');
+
+    }
+
+    public function store(Request $request)
+    {
+       
+        $attributes = request()->validate([
```

Comment:
your can access the `request()` and `Request $request` are the same so to eliminate redundancy please eliminate one of them